src/main/java/com/epam/spring/homework3/controller/HotelController.java

```diff
@@ -0,0 +1,55 @@
+package com.epam.spring.homework3.controller;
+
+
+import com.epam.spring.homework3.controller.dto.HotelDTO;
+import com.epam.spring.homework3.service.HotelService;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@Slf4j
+@RestController
+@RequiredArgsConstructor
+public class HotelController {
+
+    private final HotelService hotelService;
+
+    @ResponseStatus(HttpStatus.OK)
+    @GetMapping("/hotel")
```

Comment:
I would recommend to use @RequestMapping("/hotel") instead of duplicate the same url in each method. Also please check other controllers for the same issue.